airbyte-integrations/connectors/source-google-sheets/README.md

```diff
@@ -1,67 +1,91 @@
-# Pypi Source
```

Comment:
Wow, the previous version here weirds me out. Was it just a completely wrong doc?

Reply:
Yes it was ...